src/sentry/options/defaults.py

```diff
@@ -411,6 +411,13 @@
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Taskbroker
+register(
+    "taskbroker.enabled",
+    default=False,
+    flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
+)
+
```

Comment:
this seems unrelated to your change

Reply:
See check here: https://github.com/getsentry/sentry/pull/81968/files#diff-866fbc2da71a688b0a929236b2c864dfa8935d4a6423321d2d97a28337c6e6e8R2435

Reply:
`options` don't work inside settings -- the database isn't available yet and so this will generate warning spam

everything else utilizes a settings boolean

Reply:
hmm though I guess symbolicator uses this somehow.  strange I really don't expect that to work